packages/styletron-engine-atomic/src/client/client.js

```diff
@@ -198,9 +198,25 @@ class StyletronClient implements StandardEngine {
     return injectStylePrefixed(this.styleCache, style, "", "");
   }
 
-  renderFontFace(fontFace: FontFaceObject): string {
-    const key = declarationsToBlock(fontFace);
-    return this.fontFaceCache.addValue(key, fontFace);
+  renderFontFace(
+    fontFace: FontFaceObject | Array<FontFaceObject | string>,
+  ): string {
+    let result = "";
+    if (Array.isArray(fontFace)) {
```

Comment:
See the above comment, I think this logic instead should go in `renderDeclarativeRules`: https://github.com/styletron/styletron/blob/e1e1ebd8489e0ecea11783dc35f8170a3f8a3f07/packages/styletron-standard/src/index.js#L50-L52